browser/ui/android/strings/android_chrome_strings.grd

```diff
@@ -1120,7 +1120,7 @@ Your Brave account may have other forms of browsing history like searches and ac
         No protection (not recommended)
       </message>
       <message name="IDS_SAFE_BROWSING_NO_PROTECTION_SUMMARY" desc="Summary for Safe Browsing no protection mode.">
-        Does not protect you against dangerous websites, downloads, and extensions. You’ll still get Safe Browsing protection, where available, in other Brave services, like Gmail and Search.
+        Does not protect you against dangerous websites, downloads, or extensions.
```

Comment:
This will not work: `browser/ui/android/strings/android_chrome_strings.grd` is our copy of Chromium's original file with some substitutions we do with regex. This file will get reset with the next Chromium version update. Perhaps @samartnik can recommend the best way to swap this to a different string on Android.

Reply:
should I close this PR, till a better solution comes up?

Reply:
The only way I see to avoid direct patching is to create new base class for `RadioButtonWithDescriptionLayout` (and make patch via bytecode) and overwrite that string there in `onFinishInflate`. @akshat-rawat probably leave this issue for our dev. Thank you!

Reply:
alright I am closing the PR.